R/get_test_data.R

```diff
@@ -21,22 +21,19 @@
 #'  get_test_data(recipe = rec, x = case_death_rate_subset)
 #' @export
 
-get_test_data <- function(recipe, x){
-  # TO-DO: SOME CHECKS OF THE DATASET
-  if (any(!(c('geo_value','time_value') %in% colnames(x)))) {
-    rlang::abort("`geo_value`, `time_value` does not exist in data")
-  }
-  ## CHECK if it is epi_df?
```

Comment:
Guess this is covered by the `is.data.frame()` check below?

Reply:
There's some weirdness that can happen here. The recipe template should have all the keys and any data needed to prep/bake the test data for the predictions. So we want to be sure that the new data has all of that. If not, this should fail.

(In other words, this has been replaced with a more stringent requirement.)